src/test/Components/CreateImageWizard/steps/Oscap/Oscap.test.tsx

```diff
@@ -206,4 +191,30 @@ describe('OpenSCAP edit mode', () => {
     const expectedRequest = oscapCreateBlueprintRequest;
     expect(receivedRequest).toEqual(expectedRequest);
   });
+
+  test('fsc and packages get populated on edit', async () => {
+    const id = mockBlueprintIds['oscap'];
+    await renderEditMode(id);
+
+    // check that the FSC contains a /tmp partition
+    const fscBtns = await screen.findAllByRole('button', {
+      name: /file system configuration/i,
+    });
+    user.click(fscBtns[0]);
+    await screen.findByRole('heading', { name: /file system configuration/i });
+    await screen.findByText('/tmp');
+    // check that the Packages contain a nftable package
+    const packagesNavBtn = await screen.findByRole('button', {
+      name: /additional packages/i,
+    });
+    user.click(packagesNavBtn);
+    await screen.findByRole('heading', {
+      name: /Additional packages/i,
+    });
+    const selectedBtn = await screen.findByRole('button', {
+      name: /Selected/i,
+    });
+    user.click(selectedBtn);
+    await screen.findByText('neovim');
```

Comment:
This changed from `nftables` to `neovim` :thinking:
I know nada about the oscap, so unsure what's happening, thus I figured I'd rather ask ;)

We should change the comment above at least I guess :)

Reply:
Yeah, might look a bit chaotic :sweat_smile: The test got disabled during work on Blueprints and in the meantime the mocks changed. This should be the object that gets returned as a mocked response from the commit that made the changes: https://github.com/osbuild/image-builder-frontend/commit/a1bdcaa54ff5e46e63b82c9c99f3a36fd523b4bc#diff-07b400aeb49d2de7ed7da9354500d974cacc41ea79688a0def4ac864b9bed1b0L17-L43

'nftables' were removed from packages together with other and replaced by 'neovim' only.

Great point about the comment, I'll update it right away!

Reply:
Ah, gotcha, thanks for explaining :orange_heart: